ports/itk/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "itk",
   "version-string": "5.1.0",
```

```suggestion
  "version-semver": "5.1.0",
```
This might be served by `version-semver` here. Please see https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md